### ISSUE TITLE
ScanJS install Problem?

### ISSUE THREAD

**Issue by jreynders** [npm/npm#6392]
From the Docs:
Run ScanJS from the command line 
```
71 72  ------------------------ 
72 73  - Install [node.js](http://nodejs.org/) 
73  -- ```npm install -g mozilla/scanjs``` 
74  -- ```scanjs -t DIRECTORY_PATH``` 
 74 +- ```scanner.js -t DIRECTORY_PATH``` 

Install Output:
C:\Windows\system32>npm install -g mozilla/scanjs
npm WARN `git config --get remote.origin.url` returned wrong result (git://github.com/mozilla/scanjs) undefined
npm WARN `git config --get remote.origin.url` returned wrong result (git://github.com/mozilla/scanjs) undefined
npm WARN `git config --get remote.origin.url` returned wrong result (git@github.com:mozilla/scanjs) undefined
npm WARN `git config --get remote.origin.url` returned wrong result (git@github.com:mozilla/scanjs) undefined
npm ERR! git clone git@github.com:mozilla/scanjs undefined
npm ERR! git clone git@github.com:mozilla/scanjs undefined
npm ERR! addLocal Could not install mozilla/scanjs
npm ERR! Error: ENOENT, stat 'C:\Windows\system32\mozilla\scanjs'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Windows_NT 6.1.7601
npm ERR! command "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "mozilla/scanjs"
npm ERR! cwd C:\Windows\system32
npm ERR! node -v v0.10.32
npm ERR! npm -v 1.4.28
npm ERR! path C:\Windows\system32\mozilla\scanjs
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     C:\Windows\system32\npm-debug.log
npm ERR! not ok code 0

C:\Windows\system32>
```

**Comment by smikes** [comment]
Is this still a problem for you?

There have been a lot of improvements to npm since 2.1.9. Can you try updating your npm installation?

To update npm on Windows, follow the instructions here: https://github.com/npm/npm/wiki/Troubleshooting#upgrading-on-windows 

We are trying to clean up older npm issues, so if we don't hear back from you within a week, we will close this issue. (Don't worry -- you can always come back again and open a new issue!)

Thanks!

**Comment by jreynders** [comment]
Seems to be a problem in npm:
C:\Program Files\nodejs>npm version
npm ERR! version Bad package.json data undefined
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "version"
npm ERR! node v0.10.35
npm ERR! npm  v2.1.17

npm ERR! Unexpected token u
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Program Files\nodejs\npm-debug.log

C:\Program Files\nodejs>

This is after I upgraded with latest MSI.  

And

C:\Program Files\nodejs>npm install https://github.com/mozilla/scanjs
npm ERR! fetch failed https://github.com/mozilla/scanjs
npm WARN retry will retry, error on last attempt: Error: fetch failed with statu
s code 406
npm ERR! fetch failed https://github.com/mozilla/scanjs
npm WARN retry will retry, error on last attempt: Error: fetch failed with statu
s code 406
npm ERR! fetch failed https://github.com/mozilla/scanjs
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs
\\node_modules\\npm\\bin\\npm-cli.js" "install" "https://github.com/mozilla/scan
js"
npm ERR! node v0.10.35
npm ERR! npm  v2.1.17

npm ERR! fetch failed with status code 406
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <http://github.com/npm/npm/issues>
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs
\\node_modules\\npm\\bin\\npm-cli.js" "install" "https://github.com/mozilla/scan
js"

**Comment by smikes** [comment]
That looks terrible..

/cc @othiym23 more Windows fun for you...

**Comment by othiym23** [comment]
The problem with `npm version` will be fixed in the next release of npm, `npm@2.2.0`, which will be out today. The second issue in @jreynders's most recent comment is git trying and failing to clone from GitHub, which is a git / network configuration issue, not an npm failure. 406 is a weird status code -- what happens if you change the install command to `npm install mozilla/scanjs`?

**Comment by othiym23** [comment]
I'm going to close this as resolved now, as `npm@latest` will be `npm@2.2.0` later today, and I haven't heard anything further about the 406 code.
